### ISSUE TITLE
Error displaying miner statistics while using NanoMiner/NodeJS

### ISSUE THREAD

**Issue by tomkha** [nimiq-network/core#403]
I'd like to test NanoMiner in NodeJS
```
nodejs index.js  \
  --dumb \
  --wallet-address "NQ07 ...." \
  --type=nano --pool=eu.sushipool.com:443 --extra-data "Mighty miner" \
  --statistics=5 --log debug \
  --network main
```
it throws an exception every time it tries to display statistics
```
(node:16341) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'get' of undefined
    at $.miner.on (/home/mat/work/nimiq/clients/nodejs/index.js:243:50)
    at NanoPoolMiner.fire (/home/mat/work/nimiq/dist/node.js:392:38)
    at NanoPoolMiner._updateHashrate (/home/mat/work/nimiq/dist/node.js:34740:14)
    at Timeout._hashrateWorker.setInterval [as _onTimeout] (/home/mat/work/nimiq/dist/node.js:34523:55)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
    at emitWarning (internal/process/promises.js:65:17)
    at emitPendingUnhandledRejections (internal/process/promises.js:109:11)
    at runMicrotasksCallback (internal/process/next_tick.js:124:9)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:16341) TypeError: Cannot read property 'get' of undefined
    at $.miner.on (/home/mat/work/nimiq/clients/nodejs/index.js:243:50)
    at NanoPoolMiner.fire (/home/mat/work/nimiq/dist/node.js:392:38)
    at NanoPoolMiner._updateHashrate (/home/mat/work/nimiq/dist/node.js:34740:14)
    at Timeout._hashrateWorker.setInterval [as _onTimeout] (/home/mat/work/nimiq/dist/node.js:34523:55)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
```

**Comment by paberr** [comment]
Indeed, the error is in: https://github.com/nimiq-network/core/blob/e8606bf1ebeb4a82e2c6ab012903b447d0008763/clients/nodejs/index.js#L242-L246

Although we have a check for Nano clients in the log, we forgot it in the actual call on `$.accounts`.

**Comment by sisou** [comment]
Check out https://github.com/nimiq-network/core/pull/386 ;)

**Comment by tomkha** [comment]
ooops :)

**Comment by styppo** [comment]
Fixed in 05ac72a35aaf6d144704fa56f243d3fee8ae0eef.
